fe/fe-core/src/main/java/org/apache/doris/nereids/stats/FilterEstimation.java

```diff
@@ -348,8 +348,16 @@ A not in (1, 2, 3, 100):
         return estimated;
     }
 
+    // Right Now, we just assume the selectivity is 1 when stats is Unknown
+    private Statistics handleUnknownCase(EstimationContext context) {
+        return context.statistics;
+    }
+
     @Override
     public Statistics visitNot(Not not, EstimationContext context) {
+        if (context.statistics.containUnknownChildStats()) {
+            return handleUnknownCase(context);
```

Comment:
why we need a function, just return context.statistics

Reply:
It helps to understand the behavior that the selectivity is 1 when there is unknow input